app/cdap/components/shared/ConfigurationGroup/index.tsx

```diff
@@ -239,6 +239,53 @@ const ConfigurationGroupView: React.FC<IConfigurationGroupProps> = ({
     setOrphanErrors(Array.from(orphanedErrors));
   }
 
+  function renderConfigurationGroupHeader(group) {
+    return (
+      <div className={classes.groupTitle}>
+        <h2 className={classes.h2Title}>{group.label}</h2>
+        {group.description && group.description.length > 0 && (
+          <small className={classes.groupSubTitle}>{group.description}</small>
+        )}
+      </div>
+    );
+  }
+
+  function renderConfigurationGroupDetails(group) {
+    return (
+      <div>
+        {group.properties.map((property, j) => {
+          if (property.show === false) {
+            return null;
+          }
+          // Hiding all plugin functions if pipeline is deployed
+          if (
+            disabled &&
+            property.hasOwnProperty('widget-category') &&
+            property['widget-category'] === 'plugin'
+          ) {
+            return null;
+          }
+
+          const errorObjs = getPropertyError(property.name);
+
+          return (
+            <PropertyRow
+              key={`${property.name}-${j}`}
+              widgetProperty={property}
+              pluginProperty={pluginProperties[property.name]}
+              value={values[property.name]}
+              onChange={handleValueChanges}
+              extraConfig={extraConfig}
+              disabled={disabled}
+              locked={lockedProperties && lockedProperties[property.name]}
+              errors={errorObjs}
```

Comment:
For future: If there's a highlighted error field in the hidden section, can we auto-expand it?

Reply:
sure, will take a note